packages/create/src/index.ts

```diff
@@ -60,7 +62,13 @@ export async function createLightNode(
 
   const store = wakuStore(options);
   const lightPush = wakuLightPush(options);
-  const filter = wakuFilter(options);
+
+  let filter: (libp2p: Libp2p) => IFilter;
```

Comment:
Let's see how it goes as you try test test unsubscribe all but I expect that filter v1 and v2 will need a different interface.